albumentations/augmentations/transforms.py

```diff
@@ -3107,3 +3108,40 @@ def apply(self, img, transforms=(), **params):
 
     def get_transform_init_args_names(self):
         return ("brightness", "contrast", "saturation", "hue")
+
+
+class Sharpen(ImageOnlyTransform):
+    """Sharpen the input image and overlays the result with the original image.
+
+    Args:
+        alpha ((float, float)): range to choose the visibility of the sharpened image. At 0, only the original image is
+            visible, at 1.0 only its sharpened version is visible. Default: (0.2, 0.5).
+        lightness ((float, float)): range to choose the lightness of the sharpened image. Default: (0.5, 1.0).
+        p (float): probability of applying the transform. Default: 0.5.
+
+    Targets:
+        image
+    """
+
+    def __init__(self, alpha=(0.2, 0.5), lightness=(0.5, 1.0), always_apply=False, p=0.5):
+        super(Sharpen, self).__init__(always_apply, p)
+        self.alpha = to_tuple(alpha, 0.0)
```

Comment:
Invalid usage of to_tuple. Valid values for `alpha` in range `[0, 1]`, but if we will provide single value, for example `alpha=0.5` we will get `alpha=(-0.5, 0.5)`.
Correct usage: `to_tuple(alpha, low=0)`

Reply:
I am wrong, it is correct usage because low the second argument.